F.....F
======================================================================
FAIL: test_fast (__main__.TestBufServer)
Client with "fast" network connection sends multiple messages that the server may receive in a single read
----------------------------------------------------------------------
Traceback (most recent call last):
  File "server_test.py", line 71, in test_fast
    self.assertTrue(cstatus == 0)
AssertionError: False is not true

======================================================================
FAIL: test_username_with_space (__main__.TestBufServer)
Client sends username with a space
----------------------------------------------------------------------
Traceback (most recent call last):
  File "server_test.py", line 31, in test_username_with_space
    self.assertTrue(cstatus == 0)
AssertionError: False is not true

----------------------------------------------------------------------
Ran 7 tests in 2.481s

FAILED (failures=2)
